src/mal_log/serialization/printf_modifiers.hpp

```diff
@@ -41,7 +41,7 @@ either expressed or implied, of Rafael Gago Castano.
   #ifndef __STDC_FORMAT_MACROS
     #define __STDC_FORMAT_MACROS 1 /* this is required for some compilers */
   #endif
-  #include <inttypes.h>
+  #include "mal_log/inttypes.h"
```

Comment:
As a personal style thing, I never use relative folders because when moving folder/file locations I like to be able to just apply a search and replace string to the whole codebase, so I guess that this should be surrounded by <> instead.

Reply:
I'd prefer to detect the visual studio version and to include the compiler's <inttypes.h> when possible.